julia/Node.jl

```diff
@@ -139,4 +139,35 @@ function countConstants(tree::Node)::Integer
     else
         return 0 + countConstants(tree.l) + countConstants(tree.r)
     end
+end
+
+# Get all the constants from a tree
```

Comment:
These two tree constant functions can work if defined earlier as well, they also do not rely on anything other than a tree and so can be considered to be a Node function just like the others